aztec-nargo/Earthfile

```diff
@@ -0,0 +1,17 @@
+VERSION 0.8
+
+run:
+    FROM ubuntu:noble
+    # Install Tini as nargo doesn't handle signals properly.
+    # Install git as nargo needs it to clone.
+    RUN apt-get update && apt-get install -y git tini && rm -rf /var/lib/apt/lists/* && apt-get clean
```

Comment:
Copied from Noir's earthfile/dockerfile.... I don't know whether we need this or not.